model/modeldecoder/v2/model.go

```diff
@@ -207,20 +257,41 @@ type contextService struct {
 	// Framework holds information about the framework used in the
 	// monitored service.
 	Framework contextServiceFramework `json:"framework"`
+	// ID holds a unique identifier for the event.
+	ID nullable.String `json:"string"`
 	// Language holds information about the programming language of the
 	// monitored service.
 	Language contextServiceLanguage `json:"language"`
 	// Name of the monitored service.
 	Name nullable.String `json:"name" validate:"maxLength=1024,pattern=patternAlphaNumericExt"`
 	// Node must be a unique meaningful name of the service node.
 	Node contextServiceNode `json:"node"`
+	// Origin contains the self-nested field groups for service.
+	Origin contextServiceOrigin `json:"origin"`
 	// Runtime holds information about the language runtime running the
 	// monitored service
 	Runtime contextServiceRuntime `json:"runtime"`
 	// Version of the monitored service.
 	Version nullable.String `json:"version" validate:"maxLength=1024"`
 }
 
+type contextServiceOrigin struct {
```

Comment:
Similar to above, https://github.com/elastic/apm/pull/470/files proposes:

 - `service.origin.name`
 - `service.origin.version`
 - `service.origin.id`